source/documentation/administration_guide/topics/Configuring_the_Red_Hat_Virtualization_Manager_to_Send_SNMP_Traps.adoc

```diff
@@ -1,46 +1,107 @@
-[[Configuring_the_Red_Hat_Enterprise_Virtualization_Manager_to_Send_SNMP_Traps]]
-=== Configuring the {virt-product-fullname} {engine-name} to Send SNMP Traps
+[id="Configuring_the_Red_Hat_Enterprise_Virtualization_Manager_to_Send_SNMP_Traps"]
+= Configuring the {virt-product-fullname} {engine-name} to Send SNMP Traps
 
-Configure your {virt-product-fullname} {engine-name} to send Simple Network Management Protocol traps to one or more external SNMP managers. SNMP traps contain system event information; they are used to monitor your {virt-product-fullname} environment. The number and type of traps sent to the SNMP manager can be defined within the {virt-product-fullname} {engine-name}.
+Configure your {virt-product-fullname} {engine-name} to send Simple Network Management Protocol (SNMP) traps to one or more external SNMP managers. SNMP traps contain system event information; they are used to monitor your {virt-product-fullname} environment. The number and type of traps sent to the SNMP manager can be defined within the {virt-product-fullname} {engine-name}.
 
-This procedure assumes that you have configured one or more external SNMP managers to receive traps, and that you have the following details:
+{virt-product-fullname} supports SNMP version 2 and version 3. SNMP version 3 supports the following security levels:
 
-* The IP addresses or fully qualified domain names of machines that will act as SNMP managers. Optionally, determine the port through which {engine-name} receives trap notifications; by default, this is UDP port 162.
+*NoAuthNoPriv*:: SNMP traps are sent without any authorization or privacy.
+*AuthNoPriv*:: SNMP traps are sent with password authorization but no privacy.
+*AuthPriv*:: SNMP traps are sent with password authorization and privacy.
+
+.Prerequisites
+
+* One or more external SNMP managers are configured to receive traps.
+
+* The IP addresses or fully qualified domain names of machines that will act as SNMP managers. Optionally, determine the port through which {engine-name} receives trap notifications. The default is UDP port 162.
 
 * The SNMP community. Multiple SNMP managers can belong to a single community. Management systems and agents can communicate only if they are within the same community. The default community is `public`.
 
 * The trap object identifier for alerts. The {virt-product-fullname} {engine-name} provides a default OID of 1.3.6.1.4.1.2312.13.1.1. All trap types are sent, appended with event information, to the SNMP manager when this OID is defined. Note that changing the default trap prevents generated traps from complying with the {engine-name}'s management information base.
 
+* An SNMP username and password.
```

Comment:
I would complement here with the other security scenarios:
An SNMP username and passphrase if security level chosen is 1.
An SNMP username and passphrase if security level chosen is 2.
An SNMP username, password and private passphrase if security level chosen is 3.

Smth like that ^

Reply:
@santos1709 Take a look now.